spec/codegen_spec.rb

```diff
@@ -5,6 +5,7 @@
 RSpec.describe 'Main Script' do
   let(:file_path) { 'client/app/components/feature.tsx' }
   let(:openai_key) { 'test_openai_key' }
+  let(:openai_org_id) { 'test_openai_org_id' }
```

Comment:
would be nice for the test to also as a default the case where the org_id isn't specified as we want to make sure this works without it too